client/src/util/range.ts

```diff
@@ -22,38 +22,48 @@ rangeFill(array, start, step) -> array
 
 */
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'arr' implicitly has an 'any' type.
-function _doFill(arr, start, step, count) {
+import type { NumericArray, TypedArray } from "../common/types/entities";
+
+function _doFill<T extends TypedArray | Array<number>>(
```

Comment:
Why not define the type signature as `T extends NumberArray`?

Reply:
```
Argument of type 'NumericArray' is not assignable to parameter of type 'NumberArray'.
  Type 'Uint8ClampedArray' is not assignable to type 'NumberArray'.
    Type 'Uint8ClampedArray' is not assignable to type 'Float32Array'.
      Types of property '[Symbol.toStringTag]' are incompatible.
        Type '"Uint8ClampedArray"' is not assignable to type '"Float32Array"'.

Reply:
Right - his just means that Uint8ClampedArray is missing from the TypedArray definition inside arraytypes.ts.  Please add it, as that was an oversight on my part...

Reply:
Done 👍